django_airavata/static/common/js/components/Uppy.vue

```diff
@@ -30,6 +30,11 @@ export default {
       type: String,
       required: true
     },
+    xhrUploadMethod: {
```

Comment:
Why was this added?

Reply:
Actually this was added from the initial changes I've been doing to make the xhr method customizable. Though it was not needed later, I didn't remove that assuming it's helpful property to have

Reply:
If this isn't needed any longer, please remove.

Reply:
Removed